debian/control

```diff
@@ -6,15 +6,14 @@ Uploaders: Matthias Klumpp <mak@debian.org>,
            Laurent Bigonville <bigon@debian.org>, Michael Biebl <biebl@debian.org>
 Build-Depends: appstream,
                appstream-util,
-               debhelper (>= 9.0.0),
-               dh-autoreconf,
+               debhelper (>= 10.4),
```

Comment:
Are these changes taken from the new version of the debian packaging, or did you make them yourself? We usually update the packaging to the new debian version first (I see 3.26 is in sid) and then apply other changes on top.